scripts/ci/metrics.sh

```diff
@@ -159,10 +159,12 @@ WHERE
 GROUP BY
     Classname,
     Name
+HAVING
+    COUNTIF(Status="failed") > 0
 ORDER BY
-    COUNT(*) DESC
+    (COUNTIF(Status="failed")/COUNT(*)) DESC
```

Comment:
Can we order by ``%``

Reply:
It's a space-padded string.. not sure we want to depend on it for ordering in case we want to change the display...